docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -487,24 +486,28 @@ spec:
 
 ##### Cluster upgrade (using create-swap-and-delete)
 
-- Triggered on any changes to Version or the InfrastructureTemplate used (the KubeadmConfiguration is treated and validated as immutable).
+- Triggered on any changes to Version or the InfrastructureTemplate
```

Comment:
We now allow changes to the KubeadmConfig (with the exception of the InitConfig and ClusterConfig), though I realize now that I need to modify the hashing to accommodate for that change.

Reply:
we can make this update when those changes land